test/testcafe/spec/PollView_spec.js

```diff
@@ -0,0 +1,50 @@
+import { RequestMock, RequestLogger } from 'testcafe';
+import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
+import SelectFactorPageObject from '../framework/page-objects/SelectAuthenticatorPageObject';
+import EnrollOVViaSMSPageObject from '../framework/page-objects/EnrollOVViaSMSPageObject';
+
+import xhrIdentifyWithPassword from '../../../playground/mocks/data/idp/idx/identify-with-password';
+import xhrSelectAuthenticatorEnroll from '../../../playground/mocks/data/idp/idx/authenticator-enroll-select-authenticator';
+import xhrSafeModepolling from '../../../playground/mocks/data/idp/idx/safe-mode-polling';
+import xhrSuccess from '../../../playground/mocks/data/idp/idx/authenticator-enroll-ov-via-sms';
+
+const identifyMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentifyWithPassword)
+  .onRequestTo('http://localhost:3000/idp/idx/identify')
+  .respond(xhrSelectAuthenticatorEnroll)
+  .onRequestTo('http://localhost:3000/idp/idx/credential/enroll')
+  .respond(xhrSafeModepolling)
+  .onRequestTo('http://localhost:3000/idp/idx/poll')
+  .respond(xhrSuccess);
+
+const requestLogger = RequestLogger(/poll/, {
+  logRequestBody: true,
+  stringifyRequestBody: true,
+});
+
+fixture('Polling');
+
+async function setup(t) {
+  const identityPage = new IdentityPageObject(t);
+  const selectFactorPageObject = new SelectFactorPageObject(t);
+
+  await identityPage.navigateToPage();  
+  return [identityPage , selectFactorPageObject];
+}  
+
+test.requestHooks(requestLogger, identifyMock)('should poll', async t => {
+  let identityPage, selectFactorPage;
+  [identityPage, selectFactorPage]  = await setup(t);
```

Comment:
Why is the `let` declaration on a separate line?

Reply:
fixed it.